src/main/java/org/fundacionjala/coding/juan/SpinWords.java

```diff
@@ -0,0 +1,48 @@
+package org.fundacionjala.coding.juan;
+
+/**
+ * Created by Administrator on 3/10/2017.
+ */
+public final class SpinWords {
+    /**
+     * Constructor private.
+     */
+    private SpinWords() {
+        //no called
+    }
+
+    /**
+     * Get the words spined if it have 5 or more letters.
+     * @param  sentence is the word or words
+     * @return the word spined.
+     */
+    public static  String spinWords(final String sentence) {
+        final int limitWord = 4;
+        if (sentence == null) {
+            return "";
+        }
+        if (sentence.length() <= limitWord) {
+            return sentence;
+        } else {
```

Comment:
this else is unnecessary.
try to define the body of else directly in the next line.